Fotogjengen/hilfling-frontend
# Feature/taheera/motives view

How to test: 

[] Go to `intern/motive` 
[] Click a motive
[] Change the motive 
[] Check that the motive changes
